iothub/device/src/DeviceClient.cs

```diff
@@ -280,11 +280,11 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
         public Task<Message> ReceiveAsync() => this.internalClient.ReceiveAsync();
 
         /// <summary>
-        /// Receive a message from the device queue using the default timeout.
+        /// Receive a message from the device queue using the cancellation token.
         /// </summary>
         /// <param name="cancellationToken">A cancellation token to cancel the operation.</param>
         /// <exception cref="OperationCanceledException">Thrown when the operation has been canceled.</exception>  
-        /// <returns>The receive message or null if there was no message until the default timeout</returns>
+        /// <returns>The receive message</returns>
```

Comment:
Did the behavior change for the scenario when the cancellation token has expired but no message was received yet (do we not return a null message now)?

Reply:
We still throw an `OperationCanceledException` on cancellation token expiry; but I would no longer be getting a null message. Is this correct?

Reply:
no, still getting NULL when cancellation token is expired. I will add that in comment to description